fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -1591,39 +1594,41 @@ private static QueryParameter parseChainedParameter(HashSet<String> resourceType
                 }
 
                 HashSet<String> modifierResourceTypeNameForResourceTypes = new HashSet<>();
```

Comment:
Any particular reason for HashSet versus Set declaration?

Reply:
changed to Set